File: Coding/coding modul 4/PRAK401-2410817110007-MUHAMMAD-HAMKA-AKBARI.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int kelipatan;
    char simbol;

    // Input: Read both kelipatan and simbol from the user in the format "3 #"
    printf("Masukkan bilangan kelipatan dan simbol (misalnya 3 #): ");
    scanf("%d %c", &kelipatan, &simbol);

    // Output: Iterate from 1 to 50
    for (int i = 1; i <= 50; i++) {
        if (i % kelipatan == 0) {
            printf("%c ", simbol);  // Print the symbol instead of the number
        } else {
            printf("%d ", i);  // Print the number if it's not a multiple
        }

        // New line every 10 numbers
        if (i % 10 == 0) {
            printf("\n");  // Print a new line after every 10 numbers
        }
    }

    return 0;
}

File: Coding/coding modul 4/PRAK401-2410817110007-MUHAMMAD-HAMKA-AKBARI.py
# Input: Read both the kelipatan (multiple) and simbol (symbol) from the same line
input_data = input("Masukkan bilangan kelipatan dan simbol (misalnya 3 #): ")

# Split the input into multiple parts
kelipatan, simbol = input_data.split()

# Convert kelipatan to an integer and keep simbol as a string
kelipatan = int(kelipatan)

# Output: Iterate through numbers 1 to 50
for i in range(1, 51):
    if i % kelipatan == 0:
        print(simbol, end=" ")  # Print the symbol instead of the number
    else:
        print(i, end=" ")  # Print the number if it's not a multiple

    # New line every 10 numbers
    if i % 10 == 0:
        print()  # Create a new line every 10 numbers
